NeoML/include/NeoML/TraditionalML/KMeansClustering.h

```diff
@@ -98,11 +99,16 @@ class NEOML_API CKMeansClustering : public IClustering {
 	// false if more iterations are needed
 	bool Clusterize( IClusteringData* data, CClusteringResult& result ) override;
 
+	// Get handler of the internal thread pool
+	IThreadPool* GetThreadPool() const { return threadPool; }
```

Comment:
There is no need for it. All the `GetThreadPool()` calls are done from the methods of this class. You can replace them with `threadPool`

Reply:
fixed